packages/theme/pages/Product.vue

```diff
@@ -179,9 +179,7 @@ export default {
     const { addItem, loading } = useCart();
     const { reviews: productReviews, search: searchReviews, addReview } = useReview('productReviews');
 
-    const product = computed(() => {
-      return products.value.products && productGetters.getFiltered(products.value.products, { master: true, attributes: context.root.$route.query })[0];
-    });
```

Comment:
Is this change needed?

Reply:
> Is this change needed?

It isn't but in other cases computed was used without return and it isn't required, so it's mostly to make it similar to other computed properties.